### ISSUE TITLE
Adding Story Header

### ISSUE THREAD

**Issue by VasilijeBursac** [UnevenSoftware/vue-insta-stories#5]
Hello guys,

Can you please provide more instructions for adding Story Headers?

I tried to use the code from the documentation, but it doesn't work as it does in the live demo... 

Instead, I just get a view like this:
![chrome_glyyELyJVM](https://user-images.githubusercontent.com/53561957/120566374-6734d000-c40f-11eb-9816-be39fa0c5558.png)
And error in console: `Error in render: "TypeError: header is not a function"`

My code:

```
<template>
    <div class="pb-5 stories-container"> 
        <Stories :stories="items" class="ig-stories" >
            <template #header>
                <p>Username</p>
            </template>
        </Stories>
    </div>
</template>
```

I tried to add a component instead of the ` <p>Username</p>`  inside the `template` element, but I get the same results...

Can you, please, at least provide the code for your demo example, just so we can see how to set Story Headers to look like they do in that live demo?

Best regards! 😄

**Comment by dnldsht** [comment]
Hi the code of the demo is available in the folder packages/example

**Comment by VasilijeBursac** [comment]
> Hi the code of the demo is available in the folder packages/example

Thanks! I didn't know about that... Unfortunately, I get the same result for story headers as I have mentioned above, even if I completely copy the code from the example...

**Comment by dnldsht** [comment]
I think is a vue 2 compatibility issue will look into that the following
days and let you know when it's fixed

**Comment by VasilijeBursac** [comment]
> I think is a vue 2 compatibility issue will look into that the following days and let you know when it's fixed

Thank you for your effort! Keep up the great work!

**Comment by dnldsht** [comment]
in the 0.9.2 versione this and #7 should be fixed, let me know if I'm right

**Comment by VasilijeBursac** [comment]
> in the 0.9.2 versione this and #6 should be fixed, let me know if I'm right

Yes! It is fixed! Thank you very much! 😄

**Comment by dnldsht** [comment]
If you are building something with vue-insta-stories that is publicly available on the internet, can I have the link? I would like to create a `Who is using this?` section in the readme

**Comment by VasilijeBursac** [comment]
> If you are building something with vue-insta-stories that is publicly available on the internet, can I have the link? I would like to create a `Who is using this?` section in the readme

I would send you the link, but we are doing it as a university project, and unfortunately, I stopped using your component on this project because I need some additional features, so I made my custom component for it (It was easier for me that way). 
**Of course, If I ever use this component for anything, I will send you the link, and also mention you in the project ReadMe...**

Thank you for everything! Best regards! 😄
